frontend/smart-kickers-game/src/components/Game/GameStatistics/NumberOfShots.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './GameStatistics.css';
+
+function NumberOfShots({ statistics }) {
+  console.log(statistics);
```

Comment:
![image](https://user-images.githubusercontent.com/46759954/189661746-fec1aa80-a4f4-4884-bc49-9ae2e020c2ab.png)

Reply:
We are now protected against undefined statistics
![file](https://user-images.githubusercontent.com/56251920/189663707-71d0bff6-bc51-498a-b4ad-a9e9b4622d8d.jpg)